nextbike/cli.py

```diff
@@ -49,10 +49,13 @@ def main(test, clean, viso, traindur, pred, traingeo, predgeo, weather):
     if weather:
         weather_data = "_weather"
     if test:
-        # df = io.read_csv("Features_Duration.csv", "output")
+        df = io.read_csv("Trips.csv", "output")
         # testing_duration_models()
```

Comment:
Features duration is needed for testing training algorithms.  Therefore leave it in but uncommented